### ISSUE TITLE
The `:example_group` key in an example group's metadata hash is deprecated.

### ISSUE THREAD

**Issue by romand** [vcr/vcr#476]
```
The `:example_group` key in an example group's metadata hash is deprecated. Use
the example group's hash directly for the computed keys and
`:parent_example_group` to access the parent example group metadata instead.
Called from
/home/romand/.rvm/gems/ruby-2.1.1@vm/gems/vcr-2.9.0/lib/vcr/test_frameworks/rspec.rb:13:in
`yield'.
```

**Comment by krainboltgreene** [comment]
Ooh hey, thanks!

**Comment by CloCkWeRX** [comment]
This seemed fixed in vcr 3.0.0?

**Comment by mcfiredrill** [comment]
Assuming this is fixed in version 3.
